HPXMLtoOpenStudio/resources/unit_conversions.rb

```diff
@@ -51,6 +51,7 @@ class UnitConversions
 
     # Temperature
     ['k', 'r'] => 1.8,
+    ['deltac', 'deltaf'] => 1.8,
```

Comment:
👍 

Can we remove the line above and update any code that still references 'k' <-> 'c' conversions? They are all almost certainly doing deltaC <-> deltaF conversions, it will be less confusing to developers. I am still seeing references in hvac.rb, waterheater.rb, and weather.rb.